saleor/account/views.py

```diff
@@ -156,3 +172,51 @@ def account_delete_confirm(request, token):
 
     return TemplateResponse(
         request, 'account/account_delete_prompt.html')
+
+
+@login_required
+def email_edit(request):
+    form = EmailChangeForm(data=request.POST or None, instance=request.user)
+
+    if request.method == 'POST' and form.is_valid():
+        cache.set(
+            str(request.user.pk) + '_email_field', form.data['user_email'],
```

Comment:
I think we should create a constant for email field cache key, that would improve the readability